app/js/components/orderEntry/AllOrders.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import Orders from './Orders';
+
+const AllOrders = () => (
+  <div className="all-orders">
+    <div className="orders-breadcrumb">
+      <h3>Patient Orders</h3>
+    </div>
+    <table className="t-orders">
+      <tbody>
+        <tr>
+          <th>Date</th>
+          <th>Details</th>
+          <th>status</th>
+          <th>Actions</th>
+        </tr>
+        <Orders />
+      </tbody>
+    </table>
+  </div>
+);
+
+export default AllOrders;
```

Comment:
I see that there is a component called `Orders` that is meant to render all the orders. Naming this component `AllOrders` is kind of confusing. How about you name it `OrdersTable` since its purpose(from what I see in the code) is to display the orders in a table.

Reply:
done